modules/apps/29-fee/types/msgs.go

```diff
@@ -44,6 +46,21 @@ func (msg MsgRegisterCounterpartyAddress) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{signer}
 }
 
+// Route implements sdk.Msg
+func (msg MsgRegisterCounterpartyAddress) Route() string {
+	return RouterKey
+}
+
+// Type implements sdk.Msg
+func (msg MsgRegisterCounterpartyAddress) Type() string {
+	return TypeMsgRegisterCounterpartyAddress
+}
+
+// GetSignBytes implements sdk.Msg.
+func (msg MsgRegisterCounterpartyAddress) GetSignBytes() []byte {
+	return sdk.MustSortJSON(AminoCdc.MustMarshalJSON(&msg))
+}
```

Comment:
I personally don't think we need these for `MsgRegisterCounterpartyAddress`. You cannot relay with a ledger

Reply:
Agreed